src/components/UserProfile/UserPropertyPref.js

```diff
@@ -0,0 +1,68 @@
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Unstable_Grid2";
+import Typography from '@mui/material/Typography';
+import List from '@mui/material/List';
+import TextListItem from './TextListItem.js';
+
+import PetsIcon from '@mui/icons-material/Pets';
+import SmokingRoomsIcon from '@mui/icons-material/SmokingRooms';
+import SmokeFreeIcon from '@mui/icons-material/SmokeFree';
+import DoNotDisturbIcon from '@mui/icons-material/DoNotDisturb';
+import MapIcon from '@mui/icons-material/Map';
+import AttachMoneyIcon from '@mui/icons-material/AttachMoney';
+import SingleBedIcon from '@mui/icons-material/SingleBed';
+import ShowerIcon from '@mui/icons-material/Shower';
+
+import PrefIcon from './PrefIcon.js';
+
+
+/**
+ * Displays the user's preferred traits for a property.
+ */
+export default function UserPropertyPref({ user }) {
+	return (
+		<Paper variant="outlined" margin={3} padding={3}>
+			<Grid container columns={2} spacing={5}>
+
+				<Grid xs={2} display="flex" alignItems="center" justifyContent="center">
+					<Typography variant="h4" align="center">
+						{user.firstName}&apos;s Property Interests:
```

Comment:
```suggestion
						{user.firstName}&apos;s Property Preferences:
```
david's and my page say this so just to keep it consistent